src/kumo-api.ts

```diff
@@ -157,14 +156,10 @@ export class KumoApi {
       return false;
     }
 
-    // On initial plugin startup, let the user know we've successfully connected.
-    if(!this.securityToken) {
-      this.log.info('Kumo API: Successfully connected to the Kumo API.');
-      // Find devices and serial numbers
-      this.devices = [];
-      this.parseChildren(data[2].children);
-      this.log.info('Number of devices found:', this.devices.length);
-    }
+    //Update devices
+    this.log.info('Kumo API: Successfully connected to the Kumo API.');
+    const newDeviceCount = this.parseChildren(data[2].children);
+    this.log.info('Number of devices found:', newDeviceCount);
```

Comment:
Refresh devices on every authentication - not only on the initial authentication